### ISSUE TITLE
Changing grid size triggers overwriting of pills

### ISSUE THREAD

**Issue by robolth** [SeedSigner/SeedPills#4]
Hi, I tried changing the grid to various sizes : 8x8, 6x6, 5x10, etc.
Every time, I get warnings within OpenSCAD:

Line 1510   $fa was assigned on line 1 but was overwritten
Line 1511   $fs was assigned on line 2 but was overwritten
etc.

It seems something is amiss within the loop, but I'm not familiar with OpenSCAD.

**Comment by SeedSigner** [comment]
Can you confirm that you're using the python code to generate the openscad modeling code, and that's what you're entering into openscad?

**Comment by robolth** [comment]
Yes, I modify the Python code to increase the numbers of rows and columns:
`first_word = 1 # SET THIS VARIABLE as the first word in the BIP39 word list for the set you are generating
number_of_columns = 6 # SET THIS VARIABLE as the number of vertical columns for the desired grid
column_counter = 1 # leave this set at "1"
number_of_rows = 5 # SET THIS VARIABLE as the number of horizontal rows for the desired grid`

Here for example I increased the number of columns to 6. Didn't change anything else.

I then run:
`python3 seedpills.py >> MyGrid.txt`

...and copy-paste the MyGrid.txt content into openscad.

For 5x5 it works perfectly, but for all the other combinations I've tried there is overwriting errors.

**Comment by PatRiot1337** [comment]
> Yes, I modify the Python code to increase the numbers of rows and columns: `first_word = 1 # SET THIS VARIABLE as the first word in the BIP39 word list for the set you are generating number_of_columns = 6 # SET THIS VARIABLE as the number of vertical columns for the desired grid column_counter = 1 # leave this set at "1" number_of_rows = 5 # SET THIS VARIABLE as the number of horizontal rows for the desired grid`
> 
> Here for example I increased the number of columns to 6. Didn't change anything else.
> 
> I then run: `python3 seedpills.py >> MyGrid.txt`
> 
> ...and copy-paste the MyGrid.txt content into openscad.
> 
> For 5x5 it works perfectly, but for all the other combinations I've tried there is overwriting errors.

Did you already find a solution? Did you try to print the full words (instead of four letters max) or why did you want to change grid size?
